src/i_joystick.c

```diff
@@ -80,6 +81,18 @@ static int joystick_y_dead_zone = 33;
 static int joystick_strafe_dead_zone = 33;
 static int joystick_look_dead_zone = 33;
 
+int use_analog = 0;
+
+static int joystick_x_sensitivity = 10;
+static int joystick_move_sensitivity = 10;
+static int joystick_look_sensitivity = 10;
+
+enum
+{
+    STICK_RESPONSE_LINEAR,
```

Comment:
Sorry, if this sounds pedantic, but if we have "linear" and "cubic", where is "squared"?

Reply:
The difference between "cubic" and "squared" response curve is small. I think we need to pick what we like best and not make too many options (I like cubic!).

Reply:
ok

Reply:
I am fine with only two choices, but maybe they should be labeled less technical then. How about "linear" and "accelerated" maybe?

Reply:
Wait a minute, is this even exposed to the user anywhere? If not, let's of course stay with the current state.